Task5/script.js

```diff
@@ -0,0 +1,26 @@
+const app = document.getElementById("js-app");
+const fragment = document.createDocumentFragment();
+
+const data = [
+  { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+];
+
+const getData = new Promise((resolve) => resolve(data));
```

Comment:
【Question】
For how to write the arrow function
```suggestion
const getData = new Promise((resolve) => {
　resolve(data);
});
```
Is {} deliberately omitted? Can it be omitted?

Reply:
This is how I understand it.
In the Arrow function expression
If there is only one argument () can be omitted.
Furthermore, if the body is a single statement, {...} representing a block can be omitted. which represents a block, can be omitted.

However, if () and {} are omitted in CodeSandbox, () will be automatically restored, so only {} is omitted in the description.

Reply:
Thank you for telling me!
I didn't know it, so I learned a lot.
The writing style changes depending on the content!